runtime/src/main/java/org/corfudb/util/Utils.java

```diff
@@ -45,6 +45,36 @@
     private static Printer printer = new Textifier();
     private static TraceMethodVisitor mp = new TraceMethodVisitor(printer);
 
+    private final static char[] hexArray = "0123456789ABCDEF".toCharArray();
+
+    /** Convert a byte array to a hex string.
+     * Source:
+     * https://stackoverflow.com/questions/9655181/
+     * how-to-convert-a-byte-array-to-a-hex-string-in-java
+     * @param bytes Byte array to convert
+     * @return      Hex string representation.
+     */
+    public static String bytesToHex(byte[] bytes) {
```

Comment:
You could use a standard library for this if you want 
Google:BaseEncoding.base16()
Apache:org.apache.commons.codec.binary.Hex.encodeHexString(byteArray)